bedrock/firefox/views.py

```diff
@@ -526,10 +526,9 @@ def get_context_data(self, **kwargs):
         pre_release_channels = ["nightly", "developer", "beta"]
         channel = detect_channel(version)
 
-        # add active_locales for hard-coded locale templates
+        # activate en-GB locale for 113.0 WNP
         locale = l10n_utils.get_locale(self.request)
-        hard_coded_templates = ["id"]
```

Comment:
Note for reviewer: this is leftover from when we had a hard coded WNP for `id` locale that was shown every release (promoting Firefox Lite). That template went away some time ago, so we no longer need the locale activation logic for it.